core-feature-pack/src/main/resources/content/bin/common.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh -x
 
-setDefaultModularJvmOptions() {
+setModularJdk() {
   $JAVA --add-modules=java.se -version > /dev/null 2>&1 && MODULAR_JDK=true || MODULAR_JDK=false
+}
+
+setDefaultModularJvmOptions() {
```

Comment:
I'm trying to prepare some automation tests for JDK11 && gc-logging but it seems that you need to call `setModularJdk` at the beginning of the `setDefaultModularJvmOptions`, right? For example, `jboss-cli.sh` doesn't work without this change:
```
[mkopecky@dhcp-10-40-4-197 bin]$ ./jboss-cli.sh 
Exception in thread "main" org.jboss.modules.ModuleLoadError: java.se
...
```

Reply:
@marekkopecky Marek, the issue is in jboss-cli.sh not in standalone.sh. I have started EAP with JDK11. When I started CLI with JDK11 it really failed but when I am starting CLI with JDK10 (or lower) it is started successfully.

Reply:
CLI is just an example as I mentioned in [my previous comment](https://github.com/wildfly/wildfly-core/pull/3496#discussion_r219422721). Same behaviour are in all other scripts (domain, add-user, etc.). CLI works correctly on wf-core master. CLI doesn't work with your PR, so your PR causes regression. 

This regression in your PR blocks automatic testing of your PR.

Reply:
Java 10 is dead now, Java 11 is the thing to use and to focus on.
Java 11 is LTS release and will stay with us for long time.

Reply:
@marekkopecky Fixed. You can try now.

Reply:
I just ran the script tests in WildFly Core on Java 11 and they passed on Linux. I've not tested Windows as I don't have Java 11 installed on my VM. I'll try to see if I can install it and test there as well.